src/Sarif/Autogenerated/Region.cs

```diff
@@ -39,24 +39,32 @@ public virtual SarifNodeKind SarifNodeKind
         /// The line number of the first character in the region.
         /// </summary>
         [DataMember(Name = "startLine", IsRequired = false, EmitDefaultValue = false)]
+        [DefaultValue(0)]
```

Comment:
>DefaultValue [](http://example.com/codeflow?start=9&length=12)

have you proven this is required? 

the value of 0 is a default for start line, so is the ignore and populate attribute sufficient here? i.e., no need to specify the zero value?